client/my-sites/stats/stats-video-details/index.jsx

```diff
@@ -1,54 +1,43 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classNames = require( 'classnames' );
+import React from 'react';
+import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
-var StatsList = require( '../stats-list' ),
-	StatsListLegend = require( '../stats-list/legend' ),
-	StatsModulePlaceholder = require( '../stats-module/placeholder' ),
-	Card = require( 'components/card' );
+import StatsList from '../stats-list';
+import StatsListLegend from '../stats-list/legend';
+import StatsModulePlaceholder from '../stats-module/placeholder';
+import StatsModuleHeader from '../stats-module/header';
+import Card from 'components/card';
 
-module.exports = React.createClass( {
+export default React.createClass( {
 	displayName: 'StatModuleVideoDetails',
 
-	data: function( nextProps ) {
-		var props = nextProps || this.props;
-
-		return props.summaryList.data;
-	},
-
 	render: function() {
-		var classes,
-			isLoading = this.props.summaryList.isLoading();
+		const isLoading = this.props.summaryList.isLoading();
 
-		classes = [
+		const classes = classNames(
 			'stats-module',
 			'is-expanded',
 			'summary',
-			'is-video-details',
 			{
 				'is-loading': isLoading,
-				'is-showing-info': this.state.showInfo,
```

Comment:
@youknowriad this is the line that was removed that is currently throwing the error